Data.Service/ProjectService.cs

```diff
@@ -0,0 +1,149 @@
+﻿using Data.Entities;
+using Data.EntitiyFrameWork;
+using Data.ViewModels;
+using Data.ViewModels.ViewModels;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Linq;
+
+namespace Data.Service
+{
+    public class ProjectService : IProjectService
+    {
+        private readonly DataBase1Context dbContext;
+        public ProjectService(DataBase1Context dbContext)
+        {
+            this.dbContext = dbContext;
+        }
+        public int Add(ProjectViewModel Entity, string UserName)
+        {
+            var project = new Project();
+            project.Title = Entity.Title;
+            project.Created = Entity.Created;
+            project.Subject = Entity.Subject;
+            project.ExpirationDate = Entity.ExpirationDate;
+
+            var userName = dbContext.AspNetUsers.Where(a => a.UserName == UserName).ToList()[0];
+            project.UserId = userName.Id;
+
+            dbContext.Projects.Add(project);
+            dbContext.SaveChanges();
+            return project.Id;
+        }
+
+        public void AddCommentary(int Id, string CommentaryText, string UserName)
+        {
+
+            var commentary = new Commentary();
+            
+            commentary.Text = CommentaryText;
+            commentary.ProjectId = Id;
+            commentary.Date = DateTime.Now;
+            commentary.UserName = UserName;
+
+            var userName = dbContext.AspNetUsers.Where(a => a.UserName == UserName).ToList()[0];
+            commentary.UserId = userName.Id;
+
+
+
+            var entity = dbContext.Projects.Find(Id);
+            commentary.Project = entity;
+
+            entity.Comments = new List<Commentary>();
+            entity.Comments.Add(commentary);
+
+            dbContext.Commentaries.Add(commentary);
+            dbContext.SaveChanges();
+        }
+
+        public void AddContent(int id, string Content, string UserName)
+        {
+            var content = new Content();
```

Comment:
Use object initializer (https://www.c-sharpcorner.com/UploadFile/mahesh/object-initializer-in-C-Sharp/)